edgelet/contrib/centos/iotedge.spec

```diff
@@ -121,7 +122,7 @@ echo "==========================================================================
 %{_unitdir}/%{name}.service
 
 # sockets
-%attr(660, %{iotedge_user}, %{iotedge_group}) %{iotedge_home}/mgmt.sock
+%attr(660, %{agentuser}, %{iotedge_group}) %{iotedge_home}/mgmt.sock
```

Comment:
We are replacing the iotedge user here, which will break current functionality for customers. Instead add an ACL for agent user for mgmt sock.